pkg/f1tv/v2/api.go

```diff
@@ -40,6 +40,7 @@ const (
 	PAGE_SHOWS         PageID = 410
 	PAGE_DOCUMENTARIES PageID = 413
 	PAGE_SEASON_20201  PageID = 1510
+	PAGE_SEASON_20202  PageID = 4319
```

Comment:
We should probably replace 2021 with 2022. It will still show up in the archive section.

And `20201` was a typo, not on purpose :)